mitosheet/src/mito/components/elements/Dropdown.tsx

```diff
@@ -453,6 +453,26 @@ const Dropdown = (props: DropdownProps): JSX.Element => {
             return null;
         }
     });
+
+    const scrollRef = useRef<NodeJS.Timer | null>(null);
+    const dropdownItemContainerRef = useRef<HTMLDivElement | null>(null);
+    const [showScrollUp, setShowScrollUp] = useState(false);
+    const [showScrollDown, setShowScrollDown] = useState(childrenToDisplay.length > 6);
```

Comment:
I think it's better to:
1. Not save these as state variables
2. Compute them on every render (just in the main body of the component - or with a helper function) - directly from the ref, as you do in the `updateScrollPosition` function. 
3. Not have to remember to call this function on scroll

As it is currently, there is a bug where if you scroll in a dropdown without using these hover buttons, they don't appear / disappear at the right times!

Reply:
Ah that'll be a little more complicated than it seems I think. The component doesn't re-render on every scroll, so the reason we'd want this is **because** it's not going to automatically re-compute that on scroll. I added a scroll handler for the function to call this same `updateScrollPosition` function because we actually need a state function to make it re-render without the hover buttons when the scrolling reaches the bottom / top.